### ISSUE TITLE
Errors keep coming!

### ISSUE THREAD

**Issue by lumbar527** [Dungeons-of-Kathallion/Bane-Of-Wargs#3]
**Describe the bug**
```
  File "path/Bane-Of-Wargs/main.py", line 740, in run
    print_zone_map(map_zone)
  File "path/Bane-Of-Wargs/main.py", line 416, in print_zone_map
    to_print = zone[zone_name]["map"]["map full"]
KeyError: 'map'
```

**To Reproduce**
1. Run Bane-Of-Wargs

**Expected behavior**
I expected there to be fewer errors.

**Save file**
N/A

**Version**
What is your version of Bane of Wargs? #2

**Additional context**
N/A

**Comment by OcelotWalrus** [comment]
That's because I changed my stuff on a personal plugin. But now it's a different game, my plugin will be the vanilla stuff. Will close this issue soon.

**Comment by OcelotWalrus** [comment]
Should be fixed by d5db44d
